src/one_dragon/base/controller/pc_screenshot/gdi_screencapper_base.py

```
@@ -0,0 +1,350 @@
+import ctypes
+from typing import Optional
+import threading
+from cv2.typing import MatLike
+from one_dragon.base.controller.pc_screenshot.screencapper_base import ScreencapperBase
+import numpy as np
+import cv2
+from cv2.typing import MatLike
```

Ruff (0.12.2)

Redefinition of unused `MatLike` from line 4

Remove definition: `MatLike`

(F811)